core/src/main/resources/SqlTranslationPatterns.properties

```diff
@@ -42,20 +44,20 @@ empty_clob.target.MariaDB =''
 empty_clob.target.PostgreSQL =''
 
 # Oracle source used a.o. in testscenarios: JdbcQueryListener, ResultSetIteratingPipe, XmlQuerySender
-sysdate.source.Oracle  =SYSDATE
-sysdate.target.H2      =SYSDATE
-sysdate.target.MS_SQL  =GETDATE()
-sysdate.target.MySQL   = SYSDATE()
-sysdate.target.MariaDB = SYSDATE()
-sysdate.target.PostgreSQL = localtimestamp
+sysdate.source.Oracle		=SYSDATE
+sysdate.target.H2			=SYSTIME
```

Comment:
Removing the MODE=Oracle appearantly causes the 'date' type to become a 'timestamp' type. This makes H2 less reliable as a 'stub' database. Is there a H2 mode that does not suffer from this problem?